src/rules/newline-after-import.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @fileoverview Rule to enforce new line after import not followed by another import.
+ * @author Radek Benkel
+ */
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+function getLineDifference(node, nextToken) {
+  return nextToken.loc.start.line - node.loc.start.line
+}
+
+function ensureNoForbiddenKeyword(context, node, tokenToInspect, tokenValue) {
+  if (getLineDifference(node, tokenToInspect) === 1
+    && tokenToInspect.type === 'Keyword' && tokenToInspect.value !== tokenValue)
```

Comment:
Role of that function is to check line difference AND if another token is something different that `import/require` - if it is different, then report an error, because only thing that can be directly in next line is either `import` or `require`.